ui/app/src/styles/theme.tsx

```diff
@@ -88,6 +88,11 @@ export const defaultThemeOptions: ThemeOptions = (function() {
         root: {
           display: 'block'
         }
+      },
+      MuiTableCell: {
+        sizeSmall: {
+          padding: '6px 10px'
```

Comment:
What's the need to customise this?

Reply:
the default one is padding: 6px 24px 6px 16px, and make no sense to have right size to 24px, this is taking so much space and I think should be equals to left side